### DISCUSSION TITLE
HW 2 Part 4 Q1

### DISCUSSION THREAD

**Discussion by liam-wal** [cooperrc/computational-mechanics#102] [General]
I am having trouble trying to increase the range of randomly generated numbers from 0:1 to 0:2*pi. I tried using a for loop but that does not work with a float value in the upper bound. 

Any help would be greatly appreciated.

**Comment by cooperrc** [comment]
You can scale random numbers with multiplication or change the range with addtion/subtraction. 

e.g. 

$x_1 = [0-1]$

$x_2 = [2-10]$

```python
x1 = rng.random(100) # 100 values
x2 = (10-2)*x1 + 2 # take the x1 variable multiplied by the range and add the lowest value
```

same with $[0-2\pi]$

```python
theta = rng.random(100)*2*np.pi
```
